clients/privacy-center/app/server-environment.ts

```diff
@@ -178,6 +178,21 @@ export const validateConfig = (
       };
     }
   }
+
+  // Validate that each hidden field in custom_privacy_request_fields has a default value
+  config.actions.forEach((action) => {
+    Object.entries(action.custom_privacy_request_fields || {}).forEach(
+      ([key, field]) => {
+        if (field.hidden && field.default_value === undefined) {
+          return {
+            isValid: false,
+            message: `Default value required for hidden field '${key}' in action '${action.policy_key}'`,
```

Comment:
This error message doesn't surface completely, but I see that's a known issue https://github.com/ethyca/fides/issues/3171

Reply:
this doesn't appear to be working—I set a value to `hidden` without a `default_value` but it still says `isValid = true`! 

I think `forEach` `return`s don't work quite as we expect them to here: https://stackoverflow.com/questions/56129153/how-to-return-a-value-from-foreach